include/mfmg/dealii_operator.hpp

```diff
@@ -122,9 +126,14 @@ class DealIISmootherOperator : public Operator<VectorType>
                          std::shared_ptr<boost::property_tree::ptree> params);
 
   virtual size_t m() const override final { return _matrix.m(); }
-
   virtual size_t n() const override final { return _matrix.n(); }
 
+  virtual size_t grid_complexity() const override final { return m(); }
+  virtual size_t operator_complexity() const override final
+  {
+    ASSERT_THROW_NOT_IMPLEMENTED();
```

Comment:
Need a return to avoid warning

Reply:
Damn, it was there! I guess I messed up when cleaning and rebasing.